services/server/server.go

```diff
@@ -534,8 +537,8 @@ func (s *Server) validateAlertManagerRules(ctx context.Context, rules string) er
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	tempFile.Close()                 //nolint:errcheck
-	defer os.Remove(tempFile.Name()) //nolint:errcheck
+	tempFile.Close()                 // nolint:errcheck
```

Comment:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

https://github.com/golangci/golangci-lint#nolint

Reply:
Argh! New IDE version, fixed.